string path_to_rosbag
---
int32 predicted_label